samoa-api/src/main/java/org/apache/samoa/evaluation/EvaluatorProcessor.java

```diff
@@ -179,6 +211,28 @@ private void addMeasurement() {
     }
   }
 
+  /**
+   * This method is used to create one line of a text file containing predictions and votes (for classification only).
+   * In case, this is the first line a header line is also added
+   */
+  private void addVote() {
+    List<Vote> votes = new Vector<>();
+    Collections.addAll(votes, evaluator.getPredictionVotes());
+    Vote[] finalVotes = votes.toArray(new Vote[votes.size()]);
+    learningCurve.setVote(finalVotes);
+    logger.debug("evaluator id = {}", this.id);
+
```

Comment:
Do we need to store in LearningCurve the information of the votes? Can we print them directly using finalVotes? learningCurve.voteEntryToString is only used here to output the votes, right?